session/pessimistic_test.go

```diff
@@ -931,22 +934,30 @@ func (s *testPessimisticSuite) TestPessimisticCommitReadLock(c *C) {
 	tk.MustExec("begin;")
 	tk.MustQuery("select * from t for update").Check(testkit.Rows("1 2"))
 	done := make(chan error)
-	start := time.Now()
 	go func() {
 		var err error
 		defer func() {
 			done <- err
 		}()
+		time.Sleep(time.Millisecond * 30)
```

Comment:
Could this sleeping be avoided instead of using channel?

Reply:
The sleep is removed, PTAL